### PR TITLE
Create bind for PdfViewPager

### PR DESCRIPTION
**Pull request by 4brunu**

### CODE REVIEW

**Comment by moljac** [comment]
@4brunu

Error:

```
Script Arguments:
  Root directory: /Users/runner/work/1/s
  Root output directory: output
  Valid package namespaces: Xamarin, Mono, SkiaSharp, HarfBuzzSharp, mdoc, Masonry, GoogleGson, AndroidEasingFunctions, Square, JakeWharton.Picasso2OkHttp3Downloader, RecyclerViewAnimators, GoogleGson

Found 1 NuGet packages to validate.
Verifying NuGet metadata of /Users/runner/work/1/s/output/RecyclerView.FastScroll.2.0.1.nupkg...
NuGet metadata validation failed for /Users/runner/work/1/s/output/RecyclerView.FastScroll.2.0.1.nupkg:
The package does not start with any of the following namespaces: [ 'Xamarin', 'Mono', 'SkiaSharp', 'HarfBuzzSharp', 'mdoc', 'Masonry', 'GoogleGson', 'AndroidEasingFunctions', 'Square', 'JakeWharton.Picasso2OkHttp3Downloader', 'RecyclerViewAnimators', 'GoogleGson' ]
```

please add Xamarin as nuget package prefix.

I tend to make everything with prefix (nuget packageid, namespaces and assemblyname)

File: Android/PdfViewPager/source/PdfViewPager/PdfViewPager.csproj [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <AssemblyName>PdfViewPager</AssemblyName>
+    <RootNamespace>PdfViewPager</RootNamespace>
+    <AndroidUseLatestPlatformSdk>False</AndroidUseLatestPlatformSdk>
+    <AndroidUseIntermediateDesignerFile>True</AndroidUseIntermediateDesignerFile>
+    <AndroidClassParser>class-parse</AndroidClassParser>
+    <AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>PdfViewPager</PackageId>
```
Comment (moljac) [inline Android/PdfViewPager/source/PdfViewPager/PdfViewPager.csproj]: Xamarin prefix please

**Review by moljac — CHANGES_REQUESTED** [review]
NugetId Xamarin Prefix

File: Android/PdfViewPager/source/PdfViewPager/PdfViewPager.csproj (hunk 1 above)
Comment (4brunu) [reply Android/PdfViewPager/source/PdfViewPager/PdfViewPager.csproj]: Done, thanks for point out where the change should be 👍

**Review by moljac — APPROVED** [review]
LGTM

**Pull request merged**